ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -55,6 +55,9 @@ service DiscoveryService {
   // A compute node can be used as an argument to the ResolveService method to
   // get the service location for a service running on that compute node.
   rpc EnumerateComputeNodes(EnumerateComputeNodesRequest) returns (EnumerateComputeNodesResponse);
+
+  // Functions like ResolveService but returns selected measurement service's address and description.
+  rpc ResolveServiceWithInformation(ResolveServiceRequest) returns (ResolveServiceWithInformationResponse);
 }
```

Comment:
- Move this below the existing `ResolveService` rpc in the file.
- Do not reuse the existing `ResolveServiceRequest` message. Follow standard conventions and create a new `ResolveServiceWithInformationRequest` message instead. This message should otherwise be a duplicate of the `ResolveServiceRequest` message.
- The discovery service should be agnostic of the type of service so don't use "measurement service's" in the documentation. Simply say the "service's". I would update the documentation to say something like:

"Similar to ResolveService except information for the service is returned in addition to the location of the service. This is useful if you want to avoid the overhead of having to call EnumerateServices to get this information as part of resolution of the service. See ResolveService for additional documentation related to resolving the service."

Reply:
Updated.